### ISSUE TITLE
Docs feedback: /clients/docker-driver/_index.md

### ISSUE THREAD

**Issue by michaelkebe** [grafana/loki#4195]
In the last few days, there was a broken version of the Docker Driver latest tag. The docker log said something like this:

    
    "level=error ts=2021-08-19T12:34:53.565778352Z caller=client.go:355 container_id=ae873636286af1f5e8efe7b927b5e93bab00b3b62ec1c85b871c1de1f314112e component=client host=XXXXX.XXXXXXXXX:3100 msg=\"final error sending batch\" status=400 error=\"server returned HTTP status 400 Bad Request (400): unexpected EOF\""

I think it must have been a problem with the docker driver latest tag. There was a broken version accessible via the latest tag.

https://vsupalov.com/docker-latest-tag/

You should change the documentation to use a specific version rather than latest.

E.g.:

    docker plugin install grafana/loki-docker-driver:2.3.0 --alias loki --grant-all-permissions

Otherwise one will never know which version they are getting.

Anyway the docker plugin architecture is not very helpful here. If you specify no alias the name becomes: `grafana/loki-docker-driver:2.3.0`. But I don't want to specify the log-driver with a specific version `2.3.0` for each container I start. Especially if I want to configure docker to use the loki docker driver for every container.

Right now I think the best option is to install the loki driver like so:

    docker plugin install grafana/loki-docker-driver:2.3.0 --alias loki --grant-all-permissions

Then I can configure docker to use the `log-driver` `loki`. Docker uses the `loki:latest` aliased plugin then, which is in our case `2.3.0`.

In case of an upgrade of the loki driver I can use these steps:

* stop containers running with the `log-driver` `loki`
* `docker plugin disable loki`
* `docker plugin upgrade --grant-all-permissions loki grafana/loki-docker-driver:2.4.0`
* restart containers

Is this the right approach? Please comment.

**Comment by michaelkebe** [comment]
Keep it open. Relevant for all docker loki plugin users.

**Comment by michaelkebe** [comment]
Keep this open.

**Comment by QuinnBast** [comment]
I just created #9802 as I also just ran into this problem.

`latest` is a 2.5 year old docker image that is somewhere between 2.0.0 and 2.2.0. However, the documentation is telling new users to install and/or upgrade their installs to use `latest`. 

This is affecting all new loki-logging-driver users until this documentation gets updated or until CI/CD pipelines get fixed to point `latest` to the newest releases.
